CHANGELOG.unreleased.md

```diff
@@ -7,6 +7,8 @@
 
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
+- [Backup] add a setting `unconditionalSnapshot` to force a snapshot even when the VM is halted (PR [#6221](https://github.com/vatesfr/xen-orchestra/pull/6221))
```

Comment:
We should indicate where this value can be set, maybe like we did for `vmMigrationConcurrency`: https://github.com/vatesfr/xen-orchestra/blob/master/CHANGELOG.md#5670-2022-01-31